Packages/Tracking/Core/Runtime/Scripts/LeapServiceProvider.cs

```diff
@@ -1218,6 +1218,11 @@ private void DetectConnectedDevice(Transform targetTransform)
                     }
                 }
 
+                if (currentDevice == null && _multipleDeviceMode == MultipleDeviceMode.Disabled)
+                {
+                    currentDevice = GetLeapController().Devices[0];
```

Comment:
IIRC GetLeapController() can return null, should this have a null check just in case?